### PR TITLE
Create speakers section

### PR DESCRIPTION
**Pull request by fabricionaweb**
Work in progress

### CODE REVIEW

File: source/partials/_speakers.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<section class="speakers">
+  <ul class="speaker-list">
+    <% data.speakers.people.each do |s| %>
```
Comment (dmitryrck) [inline source/partials/_speakers.html.erb]: `s`? Rename it to `speaker` 😤

Comment (fabricionaweb) [reply source/partials/_speakers.html.erb]: hahahauuha but rubyistas don't like this?

Comment (dmitryrck) [reply source/partials/_speakers.html.erb]: short name is bad for any programming language :P

**Comment by abnersajr** [comment]
Make some layout improvements in speakers order and GitHub links.

**Pull request merged**